core/src/com/unciv/models/ruleset/validation/RulesetErrorList.kt

```diff
@@ -60,4 +91,21 @@ class RulesetErrorList : ArrayList<RulesetError>() {
                         // out of place. Prevent via kludge:
                         it.text.replace('<','〈').replace('>','〉')
                 }
+
+    companion object {
+        private val empty = RulesetErrorList()
+
+        fun of(ruleset: Ruleset? = null, sourceObject: IHasUniques? = null, vararg errors: RulesetError): RulesetErrorList {
+            if (errors.isEmpty()) return empty
```

Comment:
This is a *Mutable* instance, meaning if someone adds to it, from then on `of()` will no longer be empty. Definite future bug.